libs/core/src/utils/mixins/props-image.ts

```diff
@@ -0,0 +1,72 @@
+// utils/mixins/props-image.ts
+import { property } from 'lit/decorators.js'
+
+import { GdsElement } from '../../gds-element'
+
+type Constructor<T = GdsElement> = new (...args: any[]) => T
+
+export interface ImageProps {
+  /**
+   * The URL of the image
+   */
+  src?: string
+
+  /**
+   * The srcset attribute for responsive images
+   * Format: "image.jpg 1x, image-2x.jpg 2x" or "image-320.jpg 320w, image-640.jpg 640w"
+   */
```

Comment:
The doc comments needs to go in the implementation for CEM analyzer to pick them up.